tests/integration/local/start_lambda/test_start_lambda.py

```diff
@@ -184,13 +184,28 @@ def test_lambda_function_raised_error(self):
 
         print(response_data)
 
+        # determine the line number of the statement that will raise the error
+        statement_raising_exception = 'raise Exception("Lambda is raising an exception")'
+        statement_raising_exception_line_no = 0
+        with open("tests/integration/testdata/invoke/main.py", "r") as f:
+            for i, line in enumerate(f):
```

Comment:
Instead of adding 1 to `i` on line 193, you can do `for i, line in enumerate(f, 1)`. Which will start `i` as 1 instead of 0. 

Not material here but just wanted to bring it up so you know for next time.

Reply:
addressed, good one